cljfmt/src/cljfmt/core.cljc

```diff
@@ -130,7 +130,8 @@
   (and (line-break? zloc) (not (line-break-next? zloc))))
 
 (defn- should-unindent? [zloc]
-  (and (indentation? zloc) (not (comment-next? zloc))))
+  (and (indentation? zloc) (not (comment-next? zloc))
+       (not (and (whitespace? zloc) (line-break-next? zloc)))))
```

Comment:
It works and doesn't break any tests even when I replace `(not (and (whitespace? zloc) (line-break-next? zloc)))` with a simple `(not (line-break-next? zloc))`, but the latter looks suspicious to me.